synapse/handlers/deactivate_account.py

```diff
@@ -96,7 +96,7 @@ def deactivate_account(self, user_id, erase_data):
         yield self.store.user_set_password_hash(user_id, None)
 
         user = UserID.from_string(user_id)
-        yield self._profile_handler.set_active(user, False)
+        yield self._profile_handler.set_active(user, False, False)
```

Comment:
Don't you think we should set the value of hide to True here?

Reply:
This point may be ignored, it is just a remark

Reply:
no, here we're not operating in hide mode, so hide should be False.